app/services/github_service.rb

```diff
@@ -11,7 +15,8 @@ def get_json(url)
 
   def conn
     Faraday.new(url: "https://api.github.com/") do |faraday|
-      faraday.headers["Authorization"] = "token #{ENV["GITHUB_TOKEN"]}"
+      #faraday.params["type"] = "owner"
```

Comment:
We can discuss but we can add this line to limit repos to only the owners repos. @jmejia any thoughts as product owner?